### PR TITLE
Revert "Milestone 3: Events"

### PR DESCRIPTION
**Pull request by dami1080**
Reverts dami1080/math-magician#3
I made a mistake merging this branch to dev without submitting it  for a review 

In this project, I added the logic needed to make the Calculator component work.
According to the project requirement, I have done the following;

- Copied the files calculate.js and operate.js into a logic/ directory in your project.
- Analyzed the files calculate.js and operate.js
- Imported the files in my Calculator.js.
- Implemented event handlers needed to use the math logic from calculate.js and operate.js in your React component.
- Practiced what I have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Tested my app in the browser.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Calculator.css';
+
```
Comment (eliasecasta) [inline src/components/Calculator.js]: - [ ] The Calculator component should import the calculate.js file from the logic directory, please do so to comply with the project requirements.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              <td className="orange-colour">+</td>
+            </tr>
+            <tr>
+              <td colSpan="2">0</td>
+              <td>.</td>
+              <td className="orange-colour">=</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
```
Comment (eliasecasta) [inline src/components/Calculator.js]: - [ ] In order to implement all calculator operations, you should implement the click event for all the buttons, please do so to comply with the project requirements (The calculator is supposed to work in this milestone)

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @dami1080, I am Elias, your reviewer for this PR

Well done so far with your project!

The calculator design is on point with the one expected, good job so far!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] The calculator should work in this milestone, please fix it in order to comply with the project requirements.
- [ ] The calculate logic import should happen inside the calculator component, please fix this in order to comply with the project requirements.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**